capytaine/io/mesh_loaders.py

```diff
@@ -997,8 +1047,10 @@ def load_NEM(filename, name=None):
     'dat': load_MAR,
     'mar': load_MAR,
     'nemoh': load_MAR,
-    'wamit': load_GDF,
-    'gdf': load_GDF,
+    # 'wamit': load_GDF,
+    # 'gdf': load_GDF,
```

Comment:
Maybe we could just replace the former `load_GDF`. Is there any good reason to keep it?